src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker

```diff
@@ -200,12 +200,12 @@ kibana_vars=(
     xpack.alerting.invalidateApiKeysTask.interval
     xpack.alerting.invalidateApiKeysTask.removalDelay
     xpack.alerting.defaultRuleTaskTimeout
-    xpack.alerting.rules.execution.timeout
-    xpack.alerting.rules.execution.ruleTypeOverrides
+    xpack.alerting.rules.run.timeout
```

Comment:
Was this a renamed config from a previous release?  If so we should include both versions to allow deprecation warnings to pass through.  Feel free to merge at your discretion

Reply:
No, these were introduced in v8.2, therefore we backport to 8.2. As the release date is tomorrow, i think it's ok.

Reply:
LGTM